draft-ietf-mls-protocol.md

```diff
@@ -982,13 +980,14 @@ struct {
     CipherSuite cipher_suite;
     HPKEPublicKey hpke_init_key;
     Credential credential;
-    Extension extensions<0..2^16-1>;
+    Extension extensions<3..2^16-1>;
```

Comment:
Each extension is at least 4 bytes long (2 for type, 2 for length), so if you want the real minimum here, it should be 12.  I'm generally not sanguine about using the TLS struct definition to enforce this sort of thing, though.

Reply:
Fixed